client/stylesheets/controls.styl

```diff
@@ -7,32 +7,32 @@
 
 #ga-map-credit-box
   position absolute
-  left 0
-  bottom 0
+  left 10px
+  bottom 10px
   font-size 10px
-  margin-right 5px
-  margin-bottom 2px
+  padding 4px
   background rgba(255,255,255,0.5)
   z-index 1
 
+.ga-main-ui
+  width 335px
+
 .ga-main-ui,.ga-results-ui
   user-select none
   position absolute
   .panel
-    background none
+    background-color rgba(255,255,255,0.75)
```

Comment:
This background works well with the satellite map, but appears hard to read (to me) on Stamen Toner.  Perhaps something like #EEE with no opacity would be better?